projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -140,7 +140,7 @@ public function add_feedback_menu() {
 		}
 
 		$slug       = 'edit.php?post_type=' . $post_type;
-		$name       = $ptype_obj->labels->menu_name;
+		$name       = __( 'Contact Form', 'jetpack' );
```

Comment:
Note that this will only change the unified menu name for Jetpack sites, which is at the moment useless because [Nav Unification is disabled for those sites](https://github.com/Automattic/wp-calypso/blob/d56adcceafc0293ef87cc6a8cca47acb7f0dab76/client/state/selectors/is-nav-unification-enabled.js#L27-L31).

Does this PR seeks to change the menu name also for other sites (Simple and Atomic) and/or when Nav Unification is disabled? In that case, I think you should change the label set during the custom post type registration: https://github.com/Automattic/jetpack/blob/8ffcd878f35e27fe23b5e4ab8cbdf85a8d1be7f7/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php#L174

Reply:
You're right. I could have sworn that changed the sub-menu on my simple site. I'll update the patch.